src/rgw/rgw_rest_s3.cc

```diff
@@ -408,14 +417,28 @@ void RGWGetUsage_ObjStore_S3::send_response()
        formatter->dump_int("SuccessfulOps", total_usage.successful_ops);
        formatter->close_section(); // total 
        formatter->close_section(); // user
-     } 
-     formatter->dump_int("TotalBytes", header.stats.total_bytes);
```

Comment:
@oritwas 
IMHO, summary is a sum of all data search from usage log. and it represent process datas
but "TotalBytes" "TotalBytesRounded" "TotalEntries" are current capacity used by user, It`s a finally result
So I think put them in "CapacityUsed" is more suitable